Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:21-alpine3.18 as builder
```

Comment:
same as above, please use node in version 20. Also make sure to use the same node everywhere. You have `node:21-alpine3.18` in dockerfile and `node:21-alpine` in pipelines